asApp/src/os/RTEMS/osdNfs.c

```diff
@@ -12,68 +12,65 @@
 /**
  * Global variables
  */
-int save_restoreNFSOK    = 1;  /* for RTEMS, NFS has been mounted before autosave starts */
-int save_restoreIoErrors = 0;  /* for accumulate the IO error numbers, when the number larger than threshold, remount NFS */
+int save_restoreNFSOK =
+    1; /* for RTEMS, NFS has been mounted before autosave starts */
+int save_restoreIoErrors = 0; /* for accumulate the IO error numbers, when the
+                                 number larger than threshold, remount NFS */
 extern volatile int save_restoreDebug;
 
 /**
  * Mount the NFS
  *
  * Pre conditions:
- *    Initialization has been done: rpcUdpInit() and nfsInit(smallPoolDepth, bigPoolDepth)
- *    Please note that when using CEXP to load the RTEMS modules, the initialization will be 
- *    done automatically
+ *    Initialization has been done: rpcUdpInit() and nfsInit(smallPoolDepth,
+ * bigPoolDepth) Please note that when using CEXP to load the RTEMS modules, the
+ * initialization will be done automatically
```

Comment:
These comment lines were previously all indented the same as a paragraph under "Pre conditions"; the new version destroys that source formatting.

Reply:
I'm now leaving the comment formatting as-is.
In first version I had to make some manual changes to preserve the comment formatting, but as you see it's prone to oversight, so I think it's better to not touch them at all.